proto/process/agent.proto

```diff
@@ -452,6 +452,8 @@ message Connection {
 	// TCP state transition counters relative to last check
 	uint32 lastTcpEstablished = 30;
 	uint32 lastTcpClosed = 31;
+
+	map<uint32, uint32> dnsErrorCount = 32;
```

Comment:
Is the key for this rcode? And if so, do we want to capture successful rcodes as well ?

Reply:
Right, the key is for `rcode`. As far as I know, the only successful `rcode` is `0` - that is how we currently distinguish failed and successful responses.

Reply:
Maybe we should call this responseCountByRcode or something similar?